opm/output/eclipse/VectorItems/well.hpp

```diff
@@ -180,6 +180,7 @@ namespace Opm { namespace RestartIO { namespace Helpers { namespace VectorItems
     namespace ZWell {
         enum index : std::vector<const char*>::size_type {
```

Comment:
Do you really need the `std::vector<const char*>::size_type` - can't you just go with the default? Anyway good if you use the "new and fancy" `enum class`

Reply:
I'd advice against making this a strong enums (i.e., `enum class`).  Those cannot be used as indices into a vector without a `static_cast` at the call site.  As for `::size_type`, that's following the pattern elsewhere and is to ensure that no unit conversion happens when we do use these as indices.